modules/indexing/src/test/java/org/apache/ignite/internal/processors/query/h2/twostep/DisappearedCacheWasNotFoundMessageSelfTest.java

```diff
@@ -61,14 +62,19 @@ public void testDisappearedCacheWasNotFoundMessage() {
             fail("No CacheException emitted.");
         }
         catch (CacheException e) {
-            assertTrue(e.getMessage(), e.getMessage().contains("Cache not found on local node"));
+            boolean exp = e.getMessage().contains("Cache not found on local node");
```

Comment:
This will not work if the exception is not thrown.

Reply:
If exception is not thrown we hit `fail("No CacheException emitted.");`. Am I missing something?

Reply:
Yep, sorry, I missed that.